Exploring_Pegasus/get_inactive_qubits.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dwave_networkx as dnx
+from dwave.system.samplers import DWaveSampler
+
+dwave_sampler_pegasus = DWaveSampler(solver={'topology__type__eq': 'pegasus'})
```

Comment:
FYI, `__eq` is implied, so unless you want to be very explicit, feel free to drop it.

Also, another way of writing this solver definition is:
```
solver={'topology.type': 'pegasus'}
```
(note the dot). Or, without string keys:
```
solver=dict(topology__type='pegasus')
```

Reply:
(Applicable in other parts of this PR as well.)

Reply:
Thanks. I'm changing all of them.